core/flow/tc_throttling.go

```diff
@@ -34,7 +34,7 @@ func (c *ThrottlingChecker) DoCheck(_ base.StatNode, acquireCount uint32, thresh
 		return nil
 	}
 	if threshold <= 0 {
-		return base.NewTokenResultBlocked(base.BlockTypeFlow, "Flow")
+		return base.NewTokenResultBlocked(base.BlockTypeFlow)
```

Comment:
Maybe we need to carry the triggered rule?

Reply:
Yes, the design regarding tc_throttling.go should be improved.